apiv1.go

```diff
@@ -263,7 +264,7 @@ func (api APIv1) NewNode(n *Node) (*APIv1Node, error) {
 		Related:     related,
 		Prev:        prev,
 		Next:        next,
-
+		Custom:		 n.Custom(),
```

Comment:
Lets keep a new line to separate the deprecated fields from the rest. It also looks like you'll need to run this through `gofmt`, but maybe this is GitHub rendering the diff oddly (the field value seems to be indented to far).

Reply:
done: 2f1acd1 - also: used gofmt for the first time 🎉